alws/utils/pulp_client.py

```diff
@@ -614,11 +614,18 @@ async def get_repo_latest_version(
         repository_data = await self.request('GET', repo_href)
         return repository_data.get('latest_version_href')
 
-    async def iter_repo_packages(self, version_href: str,
-                                 limit: int = 1000, fields=None):
+    async def iter_repo_packages(
+        self,
+        version_href: str,
+        limit: int = 1000,
+        fields=None,
+        search_params: dict = None,
+    ):
         payload = {'repository_version': version_href, 'limit': limit}
         if fields is not None:
             payload['fields'] = fields
+        if search_params is not None:
```

Comment:
I'd prefer `if search_params` statement, but it's minor thing

Reply:
fixed